content/kubermatic/main/tutorials-howtos/cluster-rbac/_index.en.md

```diff
@@ -0,0 +1,108 @@
++++
```

Comment:
We already have https://docs.kubermatic.com/kubermatic/v2.21/architecture/role-based-access-control/ - while that refers to KKP RBAC and not Kubernetes RBAC, I wonder if we can at least link from that page to this to ensure we redirect people looking for Kubernetes RBAC properly.